src/memefactory/ui/components/app_layout.cljs

```diff
@@ -141,24 +141,44 @@
   ([items active-page] (app-menu items active-page 0))
   ([items active-page depth]
    (let [active-account @(subscribe [:district.ui.web3-accounts.subs/active-account])]
-     [:ul.node {:key (str depth)}
-      (doall
-       (map-indexed (fn [idx {:keys [:text :route :params :query :class :children :needs-account?]}]
-                      [:li.node-content {:key (str depth "-" idx)}
-                       [:div.item
-                        {:class (str (when class (name class))
-                                     (when (= active-page route) " active")
-                                     (when (and needs-account? (not active-account)) " disabled"))}
-                        [nav-anchor (merge
-                                     {:disabled true}
-                                     (when-not (and needs-account? (not active-account))
-                                       {:route route
-                                        :params params
-                                        :query query}))
-                         text]]
-                       (when children
-                         [app-menu children active-page (inc depth)])])
-                    items))])))
+     [:<>
+      ;; Account Balance Header for Mobile
+      (when (and active-account (= depth 0))
+        [:div.accounts
+         [:div.dank-section
+          [:div.dank-logo
```

Comment:
Please make logos defined in CSS. This change goes for app-bar as well. https://github.com/district0x/memefactory/blob/eafabdc0b82d4bd0b970a0d5ee63ea609c29ca64/src/memefactory/ui/components/app_layout.cljs#L108

Reply:
Also, good would be if you could make the balance section into reagent component. So it's reused just as 1 line.